### PR TITLE
fix: improve acceptance tests related to last changes and features

### PR DESCRIPTION
**Pull request by jesuspg**
### REVIEWERS
@hmunfru 
@flopezag 

### DESCRIPTION 
Improve old acceptant  test with behave framework

### TESTING
Local testing with dev vm and with jenkins CI

### CODE REVIEW

File: test/acceptance/aiakos_client/client.py [hunk 1]
```diff
@@ -53,13 +53,20 @@ def __init__(self, protocol, host, port, base_resource):
...
 
-        token = auth.init_auth()
-        self.headers = {"X-Auth-Token": token}
+        self.headers = {}
 
         set_representation_headers(self.headers,
                                    content_type=HTTP_REPRESENTATION_HEADER_TEXT,
                                    accept=HTTP_REPRESENTATION_HEADER_TEXT)
 
+    def add_token(self):
```
Comment (flopezag) [inline test/acceptance/aiakos_client/client.py]: This means that you do not add the token to the current call?

File: test/acceptance/features/upload_support_key.feature [hunk 2]
```diff
@@ -41,7 +41,6 @@ Feature: Upload Support keys to Aiakos Web Service: SSH and GPG Keys.
     Given the web server running properly
     When  I upload the <key_type> key for the node "qaregion2" with the content of the key "malformed"
     Then  I receive a HTTP "400" BAD REQUEST response
-    And   the file with the <key_type> key is not stored in the server
```
Comment (flopezag) [inline test/acceptance/features/upload_support_key.feature]: Why to delete this checking operation?

File: test/acceptance/aiakos_client/client.py (hunk 1 above)
Comment (jesuspg) [inline test/acceptance/aiakos_client/client.py]: only is called for post operation

File: test/acceptance/features/upload_support_key.feature (hunk 2 above)
Comment (jesuspg) [inline test/acceptance/features/upload_support_key.feature]: It's not necessary.

File: test/acceptance/aiakos_client/client.py (hunk 1 above)
Comment (flopezag) [inline test/acceptance/aiakos_client/client.py]: NTC

File: test/acceptance/features/upload_support_key.feature (hunk 2 above)
Comment (flopezag) [inline test/acceptance/features/upload_support_key.feature]: Well you are right.

NTC

**Comment by flopezag** [comment]
Please complete the description of the PR correctly.

**Pull request merged**